provider/tests/go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/pulumi/pulumi-command/provider v0.0.0-00010101000000-000000000000
 	github.com/pulumi/pulumi-go-provider v0.12.1-0.20230911215630-ea49c5f9e9b1
 	github.com/pulumi/pulumi-go-provider/integration v0.10.0
-	github.com/pulumi/pulumi/sdk/v3 v3.79.0
+	github.com/pulumi/pulumi/sdk/v3 v3.81.0
```

Comment:
Updates riding along, always best as separate PRs but I understand the expediency.